packager/app/manifest_flags.cc

```diff
@@ -37,3 +37,8 @@ ABSL_FLAG(std::string,
           "",
           "Same as above, but this applies to text tracks only, and "
           "overrides the default language for text tracks.");
+ABSL_FLAG(bool,
+          force_cl_index,
+          true,
+          "Force the muxer to order streams in the order given "
```

Comment:
update comment

Reply:
I'm thinking a mention of the old behavior should be in the release notes and not in this flag description.